### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
N/A

**Summary**
Update dependencies to not be locked.

**Does this PR introduce a breaking change?**
No

**Other information**
Another attempt at reducing deps.

### CODE REVIEW

**Comment by rishabh3112** [comment]
Lodash remains unaffected after unlocking deps as well.

File: packages/init/package.json [hunk 1]
```diff
@@ -10,10 +10,7 @@
   },
   "dependencies": {
     "@webpack-cli/generators": "^1.0.1-rc.0",
-    "@webpack-cli/utils": "^1.0.1-rc.0",
-    "colorette": "^1.2.1",
-    "jscodeshift": "0.7.0",
-    "p-each-series": "2.1.0"
+    "@webpack-cli/utils": "^1.0.1-rc.0"
```
Comment (evilebottnawi) [inline packages/init/package.json:13]: Why removed from this?

Comment (rishabh3112) [reply packages/init/package.json:13]: They weren't used by init package directly.

**Review by evenstensberg — APPROVED** [review]
Nice, let's update and fix any regressions

**Comment by rishabh3112** [comment]
Should I continue working on it? as generators break on unlocking + install size is almost unaffected.

**Comment by evilebottnawi** [comment]
@rishabh3112 Yes, there is situation when our deps will be vulnerable and to avoid unnecessary patch release we should use `^` everywhere

**Comment by rishabh3112** [comment]
Blocked by https://github.com/facebook/jest/pull/10502 to get released

**Comment by evilebottnawi** [comment]
@rishabh3112 Can you clarify?

**Comment by rishabh3112** [comment]
> @rishabh3112 Can you clarify?

`console.Console` is unavailable in jest environment while testing, thus CI fails.

**Comment by rishabh3112** [comment]
https://github.com/facebook/jest/issues/10282 for more info

**Comment by evilebottnawi** [comment]
@rishabh3112 Can you give me a link on broken tests, it is blocker for the stable release

**Comment by rishabh3112** [comment]
Yes, here are files
- `packages/generators/__tests__/utils/languageSupport.test.ts`
- `packages/generators/__tests__/utils/styleSupport.test.ts`
- `packages/generators/__tests__/addon-generator.test.ts`

The problem is `yeoman-environment` creates new environment for a generator. But when it's `.createEnv()` function is called, `new console.Console()` is called in a nested file which leads to this.

**Comment by evilebottnawi** [comment]
@rishabh3112 Let's skip them with todo, manually testing and solve it late

**Comment by rishabh3112** [comment]
/cc @evilebottnawi 

Tests are passing now.
Tested skipped tests manually, haven't emulated unit tests manually though but the major dependent features (e.g. init) are working.

**Comment by evilebottnawi** [comment]
@rishabh3112 looks good, I am afraid init can be broken, so I want to check it manually

**Comment by rishabh3112** [comment]
I have checked `init` manually, it is working fine for me.

**Review by evilebottnawi — APPROVED** [review]
/cc @webpack/cli-team

**Review by snitin315 — APPROVED** [review]

**Comment by rishabh3112** [comment]
CI is green, merging.

**Pull request merged**